pkg/server/log.go

```diff
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"io"
+	"log"
+	"strings"
+)
+
+type FilteredWriter struct {
+	io.Writer
+}
+
+var msgsToFilter = []string{
```

Comment:
Maybe, but only on server side.
IMO, that log message (`remote error: tls: bad certificate`) doesn't help too much.